Automattic/wp-calypso
# Display a warning when the IG account is not connected to a Facebook page

Related to 1203820938137329-as-1204561445874880/f

## Proposed Changes

* Display a warning when the IG account is not connected to a Facebook page

## Testing Instructions

* Ensure that you have D111103-code applied to your sandbox
* Add the blog sticker to your site as mentioned in D111103-code
* Point `public-api.wordpress.com` to your sandbox
* Goto `/marketing/connections/:site`
* Try to connect an Instagram account that is NOT linked to a Facebook page
* Confirm that you see the warning `'No Instagram business accounts linked to your Facebook pages found.'`
* Now try to connect with an Instagram account that IS connected to a Facebook page
* Confirm that you do not see the warning
* Try to disconnect the account
* Confirm that you do not see the warning

<img width="1285" alt="Screenshot 2023-06-01 at 10 35 45 AM" src="https://github.com/Automattic/wp-calypso/assets/18226415/ad2f6b0e-e919-40f0-818f-fec0610e02eb">



## Pre-merge Checklist
